content/blog/2016/2016-11-21-serverless-ci-hyper-sh-jenkins-plugin.adoc

```diff
@@ -0,0 +1,118 @@
+---
+layout: post
+title: "Serverless CI with the new Hyper.sh plugin for Jenkins"
+tags:
+- jenkins
+- docker
+- containers
+- serverless
+- immutable servers
+- CI
+- CD
+author: jmknyc06
+---
+
+= Serverless CI
+
+== What is Hyper.sh?
+
+https://hyper.sh/docker-hosting[Hyper.sh] is a container-native cloud service. Different from traditional IaaS and CaaS providers, Hyper_ does not employ a VM cluster. Instead containers are the only object the users work with.
+
+Hyper.sh’s secret sauce is its container runtime technology, which combines VM-level isolation with the speed and immutability of a container, making using and deploying containers to production as simple as using Docker on your laptop.
+
+== Why run Jenkins on Hyper.sh?
+
+Jenkins is the market leader for CI/CD workloads. The two most popular plugins are currently for AWS and DigitalOcean. The Jenkins Amazon EC2 Plugin allows users to provision EC2 instances as Jenkins Slave Nodes, while DigitalOcean Plugin allows the user to provision Droplets as Jenkins Slave Nodes. Both are VM focused solutions. +
+
+The Hyper.sh plugin allows users to launch containerized on-demand Jenkins build slaves. This presents a number of benefits:
+
+ * Lower Cost: per second billing vs. hourly pricing with AWS and Digital Ocean
+ * Faster start times: 3~10 seconds vs. 55s on Digital Ocean
+ * Compatibility: no changes to your existing workflow
+ * Easy to use: Seamless integration with Jenkins
```

Comment:
FWIW, I don't think these last two bullet points are different than what one could save about the EC2 plugin or Digital Ocean plugin for that matter, they're all using the same extension points underneath the hood.